### PR TITLE
feat/index

### PR DESCRIPTION
**Pull request by azdanov**
New look for index page.

Will be updating later on. This will do for now.

### CODE REVIEW

File: src/css/main.css [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
-  --blue: #5f60ff;
-  --teal: #74eedc;
-  --green: #8afca5;
-  --mint: #f2fdfb;
+  --blue: rgb(95, 96, 255);
+  --blue-opaque: rgba(95, 96, 255, 0.8);
+  --teal: rgb(116, 238, 220);
+  --green: rgb(138, 252, 165);
+  --mint: rgb(242, 253, 251);
 }
```
Comment (jorgegonzalez) [inline src/css/main.css]: What are the benefits or switching to RGB? I prefer hex.

Comment (azdanov) [reply src/css/main.css]: For me it was to make `--blue-opaque` color easier.

There is hex with alpha (`#5f60ff10`), but I have no clue how `0.8` is represented in hex.

And other colors are just for uniformity and maybe future reuse.

Comment (azdanov) [reply src/css/main.css]: Maybe can convert to `SCSS`, and use hex + [opacify](http://sass-lang.com/documentation/Sass/Script/Functions.html#opacify-instance_method)?

File: src/css/main.css [hunk 2]
```diff
@@ -56,12 +82,24 @@ code {
   background-color: var(--blue);
 }
 
-code span.highlight {
-  color: #fff;
-  background-color: var(--blue);
+.backdrop {
+  width: 100vw;
+  height: 100vh;
+  background: linear-gradient(170deg, #5f60ff 0%, #74eedc 50%, #8afca5 100%);
```
Comment (jorgegonzalez) [inline src/css/main.css]: Make it more blue and less green.

File: src/css/main.css (hunk 1 above)
Comment (jorgegonzalez) [reply src/css/main.css]: Use `rgba` for that single color, but let's stick to hex for the most part.

File: src/pages/index.js [hunk 3]
```diff
@@ -1,40 +1,121 @@
...
+
+  onPackageManagerChange = event => {
+    event.currentTarget.blur();
+    this.setState({ packageManager: event.currentTarget.textContent });
+  };
+
+  choosePhrase = () => {
+    const { packageManager } = this.state;
+
+    let phrase = "npm install -g zoe";
```
Comment (jorgegonzalez) [inline src/pages/index.js]: Might as well support `pnpm` too.

File: src/components/footer.js [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
 
 export default () => (
-  <footer className="ph3 ph4-ns pv3 silver bg-black-80">
-    <p className="f6 db b lh-solid mv2">
-      © {new Date().getFullYear()}{" "}
-      <a className="link dim silver" href="https://github.com/jorgegonzalez">
+  <footer className="ph3 absolute bottom-0 left-0 right-0 ph4-ns pv3">
+    <p className="f6 db b tc lh-solid mv2 black-70">
+      © {new Date().getFullYear()}
+      {" · "}
```
Comment (jorgegonzalez) [inline src/components/footer.js]: Remove this dot.

File: src/css/main.css [hunk 5]
```diff
@@ -56,12 +82,24 @@ code {
...
 }
 
 @media only screen and (min-width: 375px) {
+  code {
+    font-size: 1.5em;
+    width: 19rem;
+  }
+}
+
+@media only screen and (min-width: 600px) {
```
Comment (jorgegonzalez) [inline src/css/main.css]: There's a little bit of extra space in the bottom of the page on mobile. Not sure why.

File: src/pages/index.js [hunk 6]
```diff
@@ -1,40 +1,121 @@
...
     return (
-      <div className="mt2">
-        <code className="pa2">
-          <CopyToClipboard onCopy={this.onCopy} text={value}>
-            <span
-              className={classNames({ highlight: hovered })}
-              onMouseEnter={this.highlightText}
-              onMouseLeave={this.highlightText}
+      <div className="mt2 inline-flex items-center">
+        <ReactTooltip
```
Comment (jorgegonzalez) [inline src/pages/index.js]: Move the tooltip a *tiny* bit to the left.

**Comment by jorgegonzalez** [comment]
I think we should put the center text on a white/light-gray background. I don't like it on the gradient.

**Comment by jorgegonzalez** [comment]
Deploy preview for *zoe-js* processing.

Built with commit d9d4529dbd81a1f959eac4a6b6c86bb9ebdac44e

https://app.netlify.com/sites/zoe-js/deploys/5b6cf10073f2cf075642254f

**Comment by azdanov** [comment]
I've added initial version for text backdrop. Any suggestions?

File: src/pages/index.js (hunk 3 above)
Comment (azdanov) [reply src/pages/index.js]: I'll try to squeeze it too. I didn't add it since it's more of a niche, and options are similar to npm.

**Comment by jorgegonzalez** [comment]
There's just something about it that still feels wrong. Working on it now.

**Comment by jorgegonzalez** [comment]
image

Should add more `margin-top` to each section on mobile

**Comment by azdanov** [comment]
I'm not quite satisfied with the footer. If the screen is large, there is a lot of white space underneath. Don't know how to fix that. Other sites fill that area with footer's color.

localhost_8000_

**Comment by azdanov** [comment]
There's some redundant code left here and there. Can be taken care with refactor to styled-components.

**Comment by jorgegonzalez** [comment]
Looks awesome. Great work 😎

**Comment by jorgegonzalez** [comment]
Oops yeah the footer needs to be fixed.

**Pull request merged**